ProjectFiles/Assets/Scripts/Game/Entity/Player.cs

```diff
@@ -57,15 +57,15 @@ public class Player {
         /// The spawn method used by the server. 
         /// </summary>
         /// <param name="spawner">Allows the player to get a new spawn location</param>
-        public void Spawn(SpawnLocations spawner) {
+        public virtual void Spawn(SpawnLocations spawner) {
```

Comment:
Do we need the virtual?

Reply:
was used a few other times aswell

Reply:
Yes, it means a method can be overrided - it's not by default

Reply:
but do we override it? I cant see it anywhere

Reply:
Nsubstitute uses overriding under the hood

Reply:
Okay just odd that it doesn't error if you take them away